### PR TITLE
154937327 remove items index

### PR DESCRIPTION
**Pull request by GabrielAfflitto**
#### What does  this PR do?  
Successfuly removes /items from the routes and fixes all tests and views that depended on the route
#### Questions:
  - Do Migrations Need to be ran?
  - Do Environment Variables need to be set?
  - Any other deploy steps?

### CODE REVIEW

File: app/views/items/show.html.erb [hunk 1]
```diff
@@ -5,7 +5,7 @@
 </head>
 <body>
 	<div class = "container">
-	<h4><%= link_to "Continue shopping", items_path %></h4>
+	<h4><%#= link_to "Continue shopping", items_path %></h4>
```
Comment (Maxscores) [inline app/views/items/show.html.erb]: @GabrielAfflitto wanna update this one to the root_path?

File: config/routes.rb [hunk 2]
```diff
@@ -26,7 +26,7 @@
 
   get '/cart', :to => 'carts#index', :as => 'cart'
 
-  resources :items, only: [:index, :show]
+  resources :items, only: [:show]
```
Comment (Maxscores) [inline config/routes.rb:29]: Did we remember to delete the item/index.html?

**Review by Maxscores — CHANGES_REQUESTED** [review]
@GabrielAfflitto looks great, made a couple comments

File: config/routes.rb (hunk 2 above)
Comment (GabrielAfflitto) [reply config/routes.rb:29]: @Maxscores I actually just did. Thanks for the reminder

File: app/views/items/show.html.erb (hunk 1 above)
Comment (GabrielAfflitto) [reply app/views/items/show.html.erb]: Updated!

**Review by Maxscores — APPROVED** [review]

**Pull request merged**